components/empty-state/empty-state-styles.js

```diff
@@ -36,18 +36,35 @@ export const emptyStateIllustratedStyles = css`
 		text-align: center;
 	}
 
-	#d2l-empty-state-description {
-		margin-bottom: 0.3rem;
+	:host([illustration-name="checklist"]) .d2l-empty-state-title {
+		margin-top: 0;
```

Comment:
Seems a little weird to have custom styles just for one of the illustration types... could the SVG itself by modified a bit instead?